utilities/file_reader.py

```diff
@@ -0,0 +1,3 @@
+def generate(file):
```

Comment:
@IssaIan you can change this function name to something that gives more information about what it does.